explorer/src/components/WorldMap.tsx

```diff
@@ -1,111 +1,80 @@
-import React, { useEffect, useState } from 'react';
-import { geoEqualEarth, geoPath } from 'd3-geo';
-import { feature } from 'topojson-client';
-import { Feature, FeatureCollection, Geometry } from 'geojson';
-// import './styles.css';
+import React from 'react';
+import { scaleLinear } from 'd3-scale';
+import { ComposableMap, Geographies, Geography } from 'react-simple-maps';
+import ReactTooltip from 'react-tooltip';
+import { Box } from '@mui/material';
+import { MainContext } from '../context/main';
+import { countriesData } from '../data/countriesData';
+import { ContentCard } from './ContentCard';
 
-const uuid = require('react-uuid');
-
-const cx = 400;
-const cy = 150;
+const geoUrl =
+  'https://raw.githubusercontent.com/zcreativelabs/react-simple-maps/master/topojson-maps/world-110m.json';
 
 export const WorldMap: React.FC = () => {
-  const [geographies, setGeographies] = useState<
-    [] | Array<Feature<Geometry | null>>
-  >([]);
-
-  const [scale, setScale] = useState<number>(200);
-  const [windowResizing, setWindowResizing] = useState<boolean>(false);
-
-  const projection = geoEqualEarth()
-    .scale(scale)
-    .translate([cx, cy])
-    .rotate([0, 0]);
-
-  const fetchGeographicData = (): void => {
```

Comment:
You don't need to add the typing for `void`, that is implicit in `const someFunction = () => { ... }`